sirepo/pkcli/job_agent.py

```diff
@@ -332,11 +333,9 @@ async def _fastcgi_op(self, msg):
         if not self.fastcgi_cmd:
             m = msg.copy()
             m.jobCmd = 'fastcgi'
-            self._fastcgi_file = 'job_cmd_fastcgi.sock'
+            self._fastcgi_file = str(pkio.py_path(cfg.fastcgi_sock_dir).join('job_cmd_fastcgi.sock'))
             self._fastcgi_msg_q = sirepo.tornado.Queue(1)
             pkio.unchecked_remove(self._fastcgi_file)
-            # Avoid OSError: AF_UNIX path too long (max=100)
```

Comment:
I would document this actually in the definitely of fastcfg_sock_dir, e.g. `directory of fastcgi socket, must be less than 50 chars` -- just to pick a random number that will be less than base file name.

Reply:
https://github.com/radiasoft/sirepo/pull/2773/commits/56c0225deec58e83d9eb23d034a73a052299b95a